service/app/types.go

```diff
@@ -100,11 +102,15 @@ func (p *Pack) BeforeCreate(tx *gorm.DB) (err error) {
 	return nil
 }
 
-func (PackSlot) TableName() string {
-	return "distribution_packslots"
+func (Collectible) TableName() string {
+	return "distribution_collectibles"
 }
 
-func (ps *PackSlot) BeforeCreate(tx *gorm.DB) (err error) {
-	ps.ID = uuid.New()
+func (p *Collectible) BeforeCreate(tx *gorm.DB) (err error) {
```

Comment:
should this be  `c` (for "collectible") instead of `p` (packslot)?

Reply:
Good catch, thank you! :)